sw/device/tests/rstmgr_cpu_info_test.c

```diff
@@ -24,176 +23,253 @@ OTTF_DEFINE_TEST_CONFIG();
  * RSTMGR CPU INFO TEST
  *  This test creates a double fault by accessing a register with
  *  a non-existing address.
- *  After the double fault, the dut gets reset by the watch dog bite,
+ *  After a single fault and then a double fault,
+ *  the dut gets reset by the watch dog bite,
  *  and the test collects / checks the cpu_info from the rstmgr.
  */
 
-// non existing address
-#define kIllegalAddr1 0x4041FFF0u
-#define kIllegalAddr2 0x40003618u
-#define kSkipComp 0x12345678u
+// Unmapped Addresses
```

Comment:
Nit: if you're going to change this comment anyway, just clarify that besides being invalid addresses they don't have any meaning, these are basically just random values, so they could also be `0xA0000000`, `0xA0000004`, `0xA0000008`, etc.

Reply:
Yeah that would be nice. Changed it.